### ISSUE TITLE
Vulnerable Axios Version Used

### ISSUE THREAD

**Issue by illahi** [alchemyplatform/alchemy-sdk-js#397]
### Summary
NPM is throwing "Axios Cross-Site Request Forgery Vulnerability" when running `npm audit`
**See: https://github.com/advisories/GHSA-wf5p-g6vw-rhxx .**

```
# npm audit report
axios  0.8.1 - 1.5.1
Severity: moderate
Axios Cross-Site Request Forgery Vulnerability - https://github.com/advisories/GHSA-wf5p-g6vw-rhxx
No fix available
node_modules/axios
  alchemy-sdk  *
  Depends on vulnerable versions of axios
  node_modules/alchemy-sdk
```
### Details
https://github.com/advisories/GHSA-wf5p-g6vw-rhxx

### Impact
Not sure of security implication with regards to alchemy-sdk.
Somewhat blocking deployments for me as "npm audit fix --force" is built into my pipeline. I have manually disabled auditing for now but this is a security risk.

_**Also reported this under the Security Advisories tab a few weeks ago but have not received any response there.**_

**Comment by papkie** [comment]
👍  It would be great if it could be fixed.

**Comment by khaq** [comment]
Please update

**Comment by thebrianchen** [comment]
Sorry for the delay! This has been fixed in `v3.1.2`
